platform/linux-generic/include-abi/odp/api/abi/event_types.h

```diff
@@ -33,9 +33,6 @@ typedef enum odp_event_type_t {
 	ODP_EVENT_BUFFER = 1,
 	ODP_EVENT_PACKET = 2,
 	ODP_EVENT_TIMEOUT = 3,
-#if ODP_DEPRECATED_API
-	ODP_EVENT_CRYPTO_COMPL = 4,
-#endif
 	ODP_EVENT_IPSEC_STATUS = 5,
```

Comment:
Would changing enum value of 'ODP_EVENT_IPSEC_STATUS' to 4 be okay? Like, currently there would be a hole for enumeration, right?

Reply:
I did not want to change the other values in case someone relies on them (even if they should not) and because I just did not see any reason for such changes. The gap should not cause any issues and I think we have similar gaps in some other enums too. But I think we could remove the gap too if there is a reason to do so.

Reply:
If we have similar gaps in other enums, I think it's okay to have the same here as well.